src/index.css

```diff
@@ -187,6 +190,11 @@ html, body, #root, #app, .full-height{
   .desktop-hide{
     display: none;
   }
+  /* arrow moves slightly to the right on hover */
+  .mailchimp-button:hover{
+    background-position: 5.25px;
+    transition: 0.1s;
```

Comment:
I'd suggest creating a transition duration var to keep it to a consistent `0.2s` across all hovers

Reply:
This transition needs to be faster than the other one because it's a less-noticeable change, but yeah. Definitely should use a variable since there are two references to 0.1